back-end/api/api/urls.py

```diff
@@ -61,6 +62,11 @@
     'get': 'retrieve'
 })
 
+inbox = views.InboxViewSet.as_view({
+    'get': 'retrieve',
+    'post': 'update',
+    'delete': 'clear'
```

Comment:
Should this be 'delete': 'destroy' instead? I think that's the method DRF viewsets use.